test/node-rpc-test.js

```diff
@@ -225,4 +239,148 @@ describe('RPC', function() {
       }
     });
   });
+
+  describe('pruneblockchain', function() {
+    const network = Network.get(NETWORK);
+    const PRUNE_AFTER_HEIGHT = network.block.pruneAfterHeight;
+    const KEEP_BLOCKS = network.KEEP_BLOCKS;
+
+    let nclient, node;
+
+    before(() => {
+      network.block.pruneAfterHeight = 10;
+      network.block.keepBlocks = 10;
+    });
+
+    after(() => {
+      network.block.pruneAfterHeight = PRUNE_AFTER_HEIGHT;
+      network.block.keepBlocks = KEEP_BLOCKS;
+    });
+
+    afterEach(async () => {
+      if (nclient && nclient.opened)
+        await nclient.close();
+
+      if (node && node.opened)
+        await node.close();
+    });
+
+    it('should fail with wrong arguments', async () => {
+      node = new FullNode(nodeOptions);
+      nclient = new NodeClient(clientOptions);
+
+      await node.open();
+
+      await assert.rejects(async () => {
+        await nclient.execute('pruneblockchain', [1]);
+      }, {
+        code: errs.MISC_ERROR,
+        type: 'RPCError',
+        message: 'pruneblockchain'
```

Comment:
At first I was like "what?" then I realized this is a help/usage message, ugh terrible. We need to fix all these: https://github.com/handshake-org/hsd/issues/598

Reply:
Yep, I agree. Maybe this just replicated btc behaviour in the past ?